src/preact/component/iframe.js

```diff
@@ -144,10 +147,14 @@ export function IframeEmbedWithRef(
         <iframe
           allow={allow}
           allowFullScreen={allowFullScreen}
-          frameborder="0"
-          loading={loading}
+          // TODO: is it frameborder or frameBorder?
+          frameBorder="0"
```

Comment:
@alanorozco: found what seemed like a couple of "real" issues here. `loading` can be an invalid value, and frameborder was typed incorrectly (needs camelCasing).

Also, is `part` a real attribute? Couldn't find it on MDN on the `<iframe>` page